src/status_im/contexts/wallet/wallet_connect/sign_message/view.cljs

```diff
@@ -0,0 +1,79 @@
+(ns status-im.contexts.wallet.wallet-connect.sign-message.view
+  (:require [quo.core :as quo]
+            [quo.foundations.colors :as colors]
+            [react-native.core :as rn]
+            [react-native.safe-area :as safe-area]
+            [status-im.common.raw-data-block.view :as raw-data-block]
+            [status-im.common.standard-authentication.core :as standard-authentication]
+            [status-im.contexts.wallet.wallet-connect.sign-message.style :as style]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]))
+
+(defn- close-sheet
+  []
+  (rf/dispatch [:navigate-back]))
+
+(defn- on-auth-success
+  [password]
+  (rf/dispatch [:hide-bottom-sheet])
+  (rf/dispatch [:wallet-connect/respond-current-session password]))
+
+;; NOTE: this will be a reusable component
+(defn- header
+  [{:keys [emoji customization-color name]}]
+  [rn/view
+   {:style {:padding-vertical 12}}
+   [quo/text
+    {:size   :heading-1
+     :weight :semi-bold}
+    [rn/view {:style {:margin-top -4}}
+     [quo/summary-tag
+      {:label "dapp placeholder"}]]
+    (i18n/label :t/wallet-connect-sign-header)
+    [rn/view {:style {:padding-top 4}}
+     [quo/summary-tag
+      {:type                :account
+       :emoji               emoji
+       :label               name
+       :customization-color customization-color}]]]])
+
+(defn view
+  []
+  (let [bottom                                   (safe-area/get-bottom)
+        address                                  (rf/sub [:wallet-connect/current-request-address])
```

Comment:
This sub `:wallet-connect/current-request-address` is only used here, but it only serves the purpose of getting the address and passing it along to the next sub.

You can instead make `:wallet-connect/current-request-display-data` (which is also only used once as well) depend on the input signal `:wallet-connect/account-details-by-address`. So you'd end up with one less sub.